src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -334,6 +334,108 @@
+      <_WasmPropRequiresFullNativeRebuild Include="WasmEmitSymbolMap=$(WasmEmitSymbolMap)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmEmitSourceMap=$(WasmEmitSourceMap)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmNativeStrip=$(WasmNativeStrip)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EnableDiagnostics=$(EnableDiagnostics)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EventSourceSupport=$(EventSourceSupport)" />
+      <_WasmPropRequiresFullNativeRebuild Include="MetricsSupport=$(MetricsSupport)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmPerformanceInstrumentation=$(WasmPerformanceInstrumentation)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmProfilers=$(WasmProfilers)" />
+      <_WasmPropRequiresFullNativeRebuild Include="InvariantGlobalization=$(InvariantGlobalization)" />
+      <_WasmPropRequiresFullNativeRebuild Include="InvariantTimezone=$(InvariantTimezone)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmSkipMissingAssemblies=$(WasmSkipMissingAssemblies)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmStripAOTAssemblies=$(WasmStripAOTAssemblies)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccVerbose=$(EmccVerbose)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccLinkOptimizationFlag=$(EmccLinkOptimizationFlag)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccCompileOptimizationFlag=$(EmccCompileOptimizationFlag)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmBitcodeCompileOptimizationFlag=$(WasmBitcodeCompileOptimizationFlag)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccFlags=$(EmccFlags)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccExtraLDFlags=$(EmccExtraLDFlags)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccExtraCFlags=$(EmccExtraCFlags)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccEnableAssertions=$(EmccEnableAssertions)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccEnvironment=$(EmccEnvironment)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmInitialHeapSize=$(WasmInitialHeapSize)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccMaximumHeapSize=$(EmccMaximumHeapSize)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EmccStackSize=$(EmccStackSize)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmIncludeFullIcuData=$(WasmIncludeFullIcuData)" />
+      <_WasmPropRequiresFullNativeRebuild Include="WasmAllowUndefinedSymbols=$(WasmAllowUndefinedSymbols)" />
+      <_WasmPropRequiresFullNativeRebuild Include="EnableAggressiveTrimming=$(EnableAggressiveTrimming)" />
+      <_WasmPropRequiresFullNativeRebuild Include="PublishTrimmed=$(PublishTrimmed)" />
+      <_WasmPropRequiresFullNativeRebuild Include="_ExtraTrimmerArgs=$(_ExtraTrimmerArgs)" />
+      <_WasmPropRequiresFullNativeRebuild Include="UsingMicrosoftNETSdkWebAssembly=$(UsingMicrosoftNETSdkWebAssembly)" />
```

Comment:
how did you arrive at this list?

it'd probably also be good to sort the list and maybe split it into wasm-specific, mono-specific and generic properties to make it easier to read